nosql-db/infinispan/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>infinispan</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: Infinispan</name>
+    <dependencies>
+        <dependency>
```

Comment:
I am not sure If `io.quarkus:quarkus-resteasy` is required, because we also have `quarkus-resteasy-jackson`

Reply:
Actually, it is `quarkus-resteasy-jackson`, that is not required. Removed it.